### PR TITLE
Fixes issues with shape of output for `reduce_logsumexp` in graph mode

### PR DESCRIPTION
**Pull request by rrkarim**
Also adds docs for `select_v2` (maybe could be removed from PR since not relevant)
addr: #35099

### CODE REVIEW

**Comment by rrkarim** [comment]
@googlebot I signed it!

File: tensorflow/compiler/mlir/tensorflow/ir/tf_generated_ops.td [hunk 1]
```diff
@@ -4783,9 +4783,47 @@ select(condition, t, e) ==> [[1, 2],
 }
 
 def TF_SelectV2Op : TF_Op<"SelectV2", [NoSideEffect]> {
-  let summary = "";
+  let summary = "Selects elements from `x` or `y`, depending on `condition`.";
```
Comment (alextp) [inline tensorflow/compiler/mlir/tensorflow/ir/tf_generated_ops.td]: This is a generated file; don't edit generated files.

I believe this is generated from the api_def file for selectv2

File: tensorflow/python/ops/math_ops.py [hunk 2]
```diff
@@ -2591,7 +2591,26 @@ def reduce_logsumexp_v1(input_tensor,
                                                 reduction_indices)
   keepdims = deprecation.deprecated_argument_lookup("keepdims", keepdims,
                                                     "keep_dims", keep_dims)
-  return reduce_logsumexp(input_tensor, axis, keepdims, name)
+  keepdims = False if keepdims is None else keepdims
```
Comment (alextp) [inline tensorflow/python/ops/math_ops.py]: Why is the whole body inlined here instead of calling the non-deprecated version?

**Review by alextp — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/math_ops.py (hunk 2 above)
Comment (rrkarim) [reply tensorflow/python/ops/math_ops.py]: Really sorry for that, was thinking it was the only solution (or helper function) yesterday at 4am. Fixing now.

File: tensorflow/compiler/mlir/tensorflow/ir/tf_generated_ops.td (hunk 1 above)
Comment (rrkarim) [reply tensorflow/compiler/mlir/tensorflow/ir/tf_generated_ops.td]: I guess I don't really understand the semantics of the ops definitions, particularly the endpoint part. Guess it is not the critical part for now. If you could explain it, I can add it in PR too. Sorry for   my lack of knowledge of the code base.

**Comment by gbaned** [comment]
@rrkarim  Can you please resolve conflicts? Thanks!

**Comment by rrkarim** [comment]
> @rrkarim Can you please resolve conflicts? Thanks!

Not sure what do you mean by resolving the conflicts. I've resolved all the issues and addressed the lack of docs.

**Comment by gbaned** [comment]
@rrkarim  File tensorflow/python/ops/math_ops.py have conflicts. To check conflicts, please click on Resolve conflicts button. Please, resolve them. Thanks!

**Comment by rrkarim** [comment]
@gbaned resolved, sorry for the delay on that.

**Review by alextp — APPROVED** [review]

**Pull request merged**